### PR TITLE
Make pageType dynamic per route

### LINKED ISSUE #1975: Vary `pageType` passed to RequestContext Provider
**Is your feature request related to a problem? Please describe.**
Following the ATI work, we now have an idea of 'page type' in simorgh, however this is currently hardcoded to be `articles` [here](https://github.com/bbc/simorgh/blob/latest/src/app/Layouts/defaultPageWrapper.jsx#L18)

**Describe the solution you'd like**
Vary this, so It can be `articles`, `frontPage` or anything else we have in future.

End goal is the following code works, returning the correct page type.
```
const { pageType } = getContext(RequestContext);
```

Likely best to pair up with someone and discuss options before sinking time into this, as its likely going to be a opinionated decision :) Make sure to include the teams in the decision :)

**Testing notes**
Regression test ATI

Dev insight: Unit tests

### PR DESCRIPTION
**Pull request by dr3**
Maybe Resolves #1975

**Overall change:** One possible solution to vary page type @sareh & @benjaminhobbs  said there may be issues with putting config in the route object, but there doesnt appear to be for this example

**Code changes:**

- Pass page type from routing

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by andrew-nowak** [comment]
I can't see any particular issue with putting extra props in the route object; it seems to be explicitly suggested as a strategy in the react-router-config docs

> Consumers are free to add any additional props they'd like to a route, you can access props.route inside the component, this object is a reference to the object used to render and match.

https://github.com/ReactTraining/react-router/tree/master/packages/react-router-config#route-configuration-shape

and I can't find any reference to issues with putting props in the route config. @sareh @benjaminhobbs were you able to find the issues you described to @dr3 ?

File: src/app/Layouts/defaultPageWrapper.jsx [hunk 1]
```diff
@@ -12,10 +12,11 @@ import getOriginContext from '../contexts/RequestContext/getOriginContext';
 import getEnv from '../contexts/RequestContext/getEnv';
 import GlobalStyle from '../lib/globalStyles';
 
-const PageWrapper = ({ bbcOrigin, children, id, service, isAmp }) => {
+const PageWrapper = ({ bbcOrigin, children, id, service, isAmp, route }) => {
```
Comment (sareh) [inline src/app/Layouts/defaultPageWrapper.jsx]: I'd suggest just passing the `pageType` value to this wrapper, instead of the whole route object.

**Comment by andrew-nowak** [comment]
"How can we test that this PR is working?"
Check out this branch, apply the following changes, and `npm run dev`. When you GET a front page eg. `/igbo`, you should see `pageType: frontPage`, and when you GET an article you should see `pageType: article`

```diff
diff --git a/src/app/containers/ATIAnalytics/index.jsx b/src/app/containers/ATIAnalytics/index.jsx
index bc975622..5ae4f7a4 100644
--- a/src/app/containers/ATIAnalytics/index.jsx
+++ b/src/app/containers/ATIAnalytics/index.jsx
@@ -10,6 +10,7 @@ import { articleDataPropTypes } from '../../models/propTypes/article';
 const ATIAnalytics = ({ data }) => {
   let pageviewParams = '';
   const { pageType, platform } = React.useContext(RequestContext);
+  console.log(`pageType: ${pageType}`);
   // can allow pageType of frontPage when FrontPageAtiParams has been created
   if (pageType !== 'article') {
     return null;
diff --git a/src/app/containers/FrontPageMain/index.jsx b/src/app/containers/FrontPageMain/index.jsx
index df534493..10f1fb0c 100644
--- a/src/app/containers/FrontPageMain/index.jsx
+++ b/src/app/containers/FrontPageMain/index.jsx
@@ -7,6 +7,7 @@ import { GhostWrapper, GridItemConstrainedLarge } from '../../lib/styledGrid';
 import { ServiceContext } from '../../contexts/ServiceContext';
 import FrontPageSection from '../FrontPageSection';
 import deepGet from '../../lib/utilities/deepGet';
+import ATIAnalytics from '../ATIAnalytics';
 
 const FrontPageMain = ({ frontPageData }) => {
   const { product, serviceLocalizedName, translations } = useContext(
@@ -27,6 +28,7 @@ const FrontPageMain = ({ frontPageData }) => {
 
   return (
     <Fragment>
+      <ATIAnalytics data={frontPageData} />
       <main role="main">
         <VisuallyHiddenText as="h1">{offScreenText}</VisuallyHiddenText>
         <GhostWrapper>
```

**Review by dr3 — COMMENTED** [review]
Lol I cant approved as I opened the PR, but LGTM

**Review by j-pendlebury — APPROVED** [review]
Makes sense, looks good 👍

**Review by DenisHdz — APPROVED** [review]
LGTM

**Comment by david-boydell** [comment]
LGTM, this is ready for merge.

**Review by sareh — APPROVED** [review]
👍 Great work, @aacn500!

**Pull request merged**